mediapipe/graphs/instantmotiontracking/calculators/BUILD

```diff
@@ -0,0 +1,97 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//mediapipe/framework/port:build_config.bzl", "mediapipe_cc_proto_library")
+
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+exports_files(["LICENSE"])
+
+mediapipe_cc_proto_library(
+    name = "sticker_buffer_cc_proto",
+    srcs = [
+        "sticker_buffer.proto"
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@com_google_protobuf//:protobuf",
+    ],
+)
+
+cc_library(
+    name = "transformations",
+    srcs = [
+        "transformations.h"
+    ],
+)
+
+cc_library(
+    name = "sticker_manager_calculator",
+    srcs = ["sticker_manager_calculator.cc"],
+    deps = [
+        ":sticker_buffer_cc_proto",
+        ":transformations",
+        "//mediapipe/framework:calculator_framework",
+        "//mediapipe/framework:timestamp",
+        "//mediapipe/framework/port:integral_types",
+        "//mediapipe/framework/port:logging",
+        "//mediapipe/framework/port:ret_check",
+        "//mediapipe/framework/port:source_location",
+        "//mediapipe/framework/port:status",
+    ],
+    alwayslink = 1,
+)
+
+cc_library(
+    name = "matrices_manager_calculator",
+    srcs = ["matrices_manager_calculator.cc"],
+    deps = [
+        ":transformations",
+        "@eigen_archive//:eigen",
+        "//mediapipe/framework:calculator_framework",
+        "//mediapipe/framework:timestamp",
+        "//mediapipe/framework/formats:detection_cc_proto",
+        "//mediapipe/framework/formats:image_frame",
+        "//mediapipe/framework/formats:image_frame_opencv",
+      	"//mediapipe/framework/port:integral_types",
+      	"//mediapipe/framework/port:logging",
+      	"//mediapipe/framework/port:opencv_imgproc",
+      	"//mediapipe/framework/port:ret_check",
+      	"//mediapipe/framework/port:source_location",
+      	"//mediapipe/framework/port:status",
+        "//mediapipe/graphs/object_detection_3d/calculators:model_matrix_cc_proto",
```

Comment:
It's still showing up as weirdly aligned for me.  Is this not showing up in your editing environment?

Reply:
In my editor, as well as Github when I click 'view file', the alignment seems to have no issues. This might be an issue with the Pull Request changes viewer, but the file itself appears to have no alignment issues.